### PR TITLE
(DS-550) ci: add pipelines workflows

### PR DESCRIPTION
**Pull request by JuanDavidBuitrago**
## Description

This PR adds CI pipelines:

- bumpversion
- commitlint
- labeler
- pull_request_template
- test

### CODE REVIEW

**Review by MaferMazu — APPROVED** [review]
This looks good to me, but I think we need to add the test workflow too. Can you add it to this PR or another?

**Comment by JuanDavidBuitrago** [comment]
> This looks good to me, but I think we need to add the test workflow too. Can you add it to this PR or another?

You're right, we need to add test workflow too. I think it's a good idea to add in this PR, but we need to add docstrings in all python files. Maybe we can create a Jira card to complete this.

**Comment by JuanDavidBuitrago** [comment]
I added the workflow for the tests and fixed the code quality

File: tests/test_distro.py [hunk 1]
```diff
@@ -9,4 +9,4 @@ def test_version():
     """
     Tutor distro plugin tests version
     """
-    assert __version__ == '15.0.0'
+    assert __version__ == '15.0.2'
```
Comment (MaferMazu) [inline tests/test_distro.py:12]: I think this test is going to be annoying to maintain. I think it would be good to eliminate it. What do you think?

**Review by MaferMazu — COMMENTED** [review]
The rest looks good to me.

File: tests/test_distro.py (hunk 1 above)
Comment (JuanDavidBuitrago) [reply tests/test_distro.py:12]: Sorry, I forget remove this, it is not useful

thanks, @MaferMazu

**Review by MaferMazu — APPROVED** [review]
Looks good to me.
The commitlint workflow already has a v5 (used in eox-core), but I think we are okay.

**Pull request merged**